README.md

```diff
@@ -23,6 +23,8 @@ Specific configuration that must be seen :
   "config": {
     ...
     "minetest-download": "${minetestDownload}",
+    "minetest-link": "${minetestLink}",
+    "minetest-wiki": "${minetestWiki}",
```

Comment:
T'as aussi la partie variable "In  your springboard, you must include these variables :..." à rajouter!